aip/adopting.md

```diff
@@ -105,6 +101,37 @@ The `tests` folder contains an npm test module that will validate your data
 files. Running these tests requires npm and mocha. Once these are installed
 that tests can be ran with the `npm test` command.
 
+## AIP Extensions
+
+Sometimes an organization within a domain may want to adopt most or all of the
+AIPs in a domain, and also need the ability to make some adjustments. This is
+considered an extension of an AIP domain, and is expected to reside within the
+domain.
+
+AIP extensions alter an AIP by providing a new file that should be displayed
+when viewing that AIP ID within the organization's context.
+
+AIP extensions are expected to follow the same rules regarding AIP numbering
+when creating custom AIPs that are specific to their context.
+
+When extending an AIP domain, an organization context **must** create a new
+folder in the `aip` directory containing all of their altered or custom AIP
+files. Additionally they must create an `index.md` file with the following
+front-matter:
+
+```
+aip_index:
+  scope: <unique scope for domain>
+  order: <number that determines order in nav>
+exclude_from_search: true
+js: /assets/js/aip/aip-index.js
+permalink: /<unique scope for domain>
+```
+
+Note that the value of `scope` will be used to identify the context under which
+AIPs are being viewed and will determine the content of the nav menu and how
+references to other AIPs are resolved.
+
```

Comment:
The aip_index is how I get and manage context.

The final solution will need to be a little more complicated than this, but a simple way to think of it is this:
1. Navigating to an aip_index file will set the context.
2. Navigating to any other file will utilize the context to determine whether or not to go to the domain level aip, or an aip extension (if it exists).